controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -980,7 +980,7 @@ public OperationContext createStreamContext(String scopeName, String streamName,
         return Futures.completeOn(streamObj.completeCommittingTransactions(record, context), executor)
                 .thenAcceptAsync(result -> {
                     streamObj.getNumberOfOngoingTransactions(context).thenAccept(count ->
-                            TransactionMetrics.reportOpenTransactions(scope, stream, count));
+                            TransactionMetrics.reportOpenTransactions(scope, stream, count.intValue()));
```

Comment:
Just FYI it is theoretically possible this will overflow, although I find it hard to believe we may have more than 2 billion open transactions per stream at any given time.